package_test.go

```diff
@@ -52,6 +52,9 @@ func TestPackageBinfile(t *testing.T) {
 		}
 		got := pkg.Binfile()
 		want := filepath.Join(ctx.Bindir(), tt.want)
+		if pkg.gotargetos == "windows" {
```

Comment:
they were supposed to feed into NewContext, via gb.GOOS, and gb.GOARCH, but they aren't hooked up yet. No matter.